### DISCUSSION TITLE
Sources used for the course

### DISCUSSION THREAD

**Discussion by gonzalo-munillag** [OpenMined/courses#159] [Feedback] [answered]
I have finished the 5 lessons and I am happy with the overall quality and indeed one learns how to look at the space in a different manner, even if one has been in the space for some time. Also what I found very cool is to get the latest opinion about tech from top experts, that is gold.

What I think people would appreciate is that you release the sources you used to compile the scripts. So people can know, for example, what is your basis for stating that the best long-term strategy is to be an endorser and architect of empowering information flows? 

Thank you from the bottom of my heart for all the work.

Best,
Gonzalo

PS: There was a post I wrote yesterday about the calculation of a privacy attack which was only answered by a fellow student and was deleted at some point today. Out of mere curiosity, was there a particular reason for the deletion? Cheers

**Comment by gonzalo-munillag** [comment]
Never mind the PS, for some reason, I just found it in the discussion mix and not in the dedicated "Feedback" forum.
Cheers!

**Reply by yemikifouly** [reply]
Hahahah... Yeah. It wasn't deleted. I just changed [its category](https://github.com/OpenMined/courses/discussions/144) to make sure our tech people see it more easily. I am checking what you said about the age right now and passing the question to someone who can help. 🙂

**Comment by hwrdtm** [comment, marked as answer]
@gonzalo-munillag this is currently being tracked in https://github.com/OpenMined/courses/issues/256! Please feel free to mark this discussion as answered, and we can discuss the details of this further in that ticket. 🙏

**Reply by gonzalo-munillag** [reply]
For sure! But I am failing to find where can I close this with "answered"

**Reply by souravcipher** [reply]
Hi @gonzalo-munillag! I have marked this thread as answered, let us know if you still have any question/feedback.

Have a nice day!! : )
